build/build-extensions.ps1

```diff
@@ -241,7 +242,11 @@ function CreateSiteExtensions() {
     
     $zipOutput = "$buildOutput\SiteExtension"
     New-Item -Itemtype directory -path $zipOutput -Force > $null
-    ZipContent $siteExtensionPath "$zipOutput\Functions.$extensionVersion$runtimeSuffix.zip"
+    if ($minorVersionPrefix -eq "10") {
+        ZipContent $siteExtensionPath "$zipOutput\Functions.$extensionVersion$runtimeSuffix.zip"
+    } else {
+		ZipContent $siteExtensionPath "$zipOutput\FunctionsInProc.$extensionVersion$runtimeSuffix.zip"
```

Comment:
i think this one is using tabs instead of spaces too... probably renders fine in your editor, but not here. See if you can fix those up?